Sources/dojo-ios-sdk/Classes/Networking/NetworkService.swift

```diff
@@ -12,7 +12,7 @@ class NetworkService: NetworkServiceProtocol {
     let session: URLSession
     let timeout: TimeInterval
     
-    required init(timeout: TimeInterval) {
+    required init(timeout: TimeInterval = 25) {
```

Comment:
Just to confirm a default timeout is intentional?

Reply:
yes